core/controlplane/config/templates/cloud-config-controller

```diff
@@ -345,11 +347,16 @@ coreos:
       command: start
       content: |
         [Unit]
-        Wants=kubelet.service docker.service
-        After=kubelet.service
+        Wants=kubelet.service docker.service install-kube-system.service
+        After=kubelet.service install-kube-system.service
 
         [Service]
-        Type=oneshot
+        Type=simple
```

Comment:
Yes, `type=simple` is better!
I'm now wondering where we actually need `Type=oneshot`. Nowhere?
Can we just transform every `Type=oneshot` service into `Type=simple` service with all the meaningful pieces in `ExecStartPre`s and `ExecStart` being `/bin/true`?